service/src/main/resources/db/changelog/changesets/20210930_rename_provider_id.yaml

```
@@ -0,0 +1,8 @@
+databaseChangeLog:
+  - changeSet:
+      id: "rename_provider_id_column"
+      author: kfeldman
+      changes:
+        - sql:
+            sql: >
+              ALTER TABLE linked_account RENAME COLUMN provider_id TO provider_name;
```

Should this just be one line?
```suggestion
            sql: ALTER TABLE linked_account RENAME COLUMN provider_id TO provider_name;
```

---

yep probably

---

actually it looks like our other one liners aren't inlined, should we be consistent?

---

we have a frustrating mix of `>`, `"`, and neither delimiting our changeset sql strings. Your call.

---

Cool cool, maybe I'll just make 'em consistent then. My inclination is to use `>` because most of the multi-line ones are already handled that way